src/Miki/Modules/DeveloperModule.cs

```diff
@@ -339,5 +343,48 @@ public async Task BanUserAsync(IContext e)
 					.BanAsync(context);
 			}
 		}
-	}
+
+        [Command("dailyedit")]
+        [RequiresScope("developer")]
+        public class DailyEditCommand
+        {
+            [Command("resettimer")]
+            //[RequiresScope("developer")]
```

Comment:
Don't commit commented code like this, instead make yourself a developer on your own database.